share.android.js

```diff
@@ -39,6 +44,21 @@ export default class Share extends Component {
     this.setState({ isOpen: false })
   }
 
+  async getArticleByUrl(url) {
+    // Token and server are not yet set into state bucause of
+    // asynchronous nature of setState.
+    const server = await RNSKBucket.get('server')
+    const token = await RNSKBucket.get('token')
+
+    return fetch(`${server}/${API_VERSION}/bookmarks?url=${encodeURIComponent(url)}`, {
+      method: 'GET',
+      headers: {
+        'Authorization': `JWT ${token}`,
+        'Accept': 'application/json',
+      },
+    }).then((response) => response.json())
```

Comment:
Consider checking whether a response is successful
(server may return json with 401)